hack/build-jsonnet.sh

```diff
@@ -22,15 +22,26 @@ while IFS= read -r line; do
 	files+=("$line")
 done < <(jq -r 'keys[]' "${TMP}/main.json")
 
-for file in "${files[@]}"
-do
-	dir=$(dirname "${file}")
-	path="${prefix}/${dir}"
-	mkdir -p "${path}"
-	# convert file name from camelCase to snake-case
-	fullfile=$(echo "${file}" | sed 's/\(.\)\([A-Z]\)/\1-\2/g' | tr '[:upper:]' '[:lower:]')
-	jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml"
+maxProc=$(nproc || echo 4)
+
+# process all files in parallel
+for file in "${files[@]}"; do
+	{
+		dir=$(dirname "${file}")
+		path="${prefix}/${dir}"
+		mkdir -p "${path}"
+
+		# convert file name from camelCase to snake-case
+		fullfile=$(echo "${file}" | sed 's/\(.\)\([A-Z]\)/\1-\2/g' | tr '[:upper:]' '[:lower:]')
+		jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml"
+	}&
```

Comment:
@jan--f this should give us the same perf as `jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml" &` with the additional benefit of easier to reason with. I.E. consider the entire block as a function.